common/src/main/java/com/amazonaws/services/schemaregistry/utils/AWSSchemaRegistryConstants.java

```diff
@@ -158,6 +158,12 @@
      */
     public static final String JACKSON_DESERIALIZATION_FEATURES = "jacksonDeserializationFeatures";
 
+    /**
+     * Name of the UserAgent application using the serializer/de-serializer library.
+     * Ex: KPL, Kafka, KafkaConnect etc.
+     */
+    public static final String USER_AGENT_APP = "userAgentApp";
```

Comment:
I would recommend naming the variables in kafka friendly format since we support kafka systems.
Change it to "user.agent.app"